docs/source/core_concepts/index.rst

```diff
@@ -8,10 +8,12 @@ We will look through the basics of Iroha in this section.
 .. WARNING:: Docs are constantly updated and this section is going to be improved.
```

Comment:
Just a stylish note: maybe, it's better to highlight the word **warning**?

Reply:
it is syntax from https://sphinx-rtd-theme.readthedocs.io/en/latest/demo/demo.html#admonitions